src/components/FeedbackMessage/FeedbackMessage.stories.tsx

```diff
@@ -0,0 +1,172 @@
+    const { info, loading, success, warning, error } = useFeedbackMessage()
+
+    const { message } = context.args
+
+    return (
+      <div style={{ display: 'flex', gap: '4px' }}>
+        <Button
+          hierarchy={Hierarchy.Neutral}
+          icon={<Icon color="blue" name="PiInfo" size={16} />}
+          onClick={() => info({ message })}
+        >
+Info
+        </Button>
+        <Button
+          hierarchy={Hierarchy.Neutral}
+          icon={<Icon color="green" name="PiCheck" size={16} />}
+          onClick={() => success({ message })}
+        >
+Success
+        </Button>
+        <Button
+          hierarchy={Hierarchy.Neutral}
+          icon={<Icon name="PiSpinner" size={16} />}
+          onClick={() => loading({ message })}
+        >
+Loading
+        </Button>
+        <Button
+          hierarchy={Hierarchy.Neutral}
+          icon={<Icon color="orange" name="PiWarning" size={16} />}
+          onClick={() => warning({ message })}
+        >
+Warning
+        </Button>
+        <Button
+          hierarchy={Hierarchy.Neutral}
+          icon={<Icon color="red" name="PiXCircle" size={16} />}
+          onClick={() => error({ message })}
+        >
+Error
+        </Button>
+      </div >
+    )
+  }],
+}
+
+export const FeedbackMessageWithExtraContent: Story = {
+  decorators: [(_Story, context) => {
+    const { message } = context.args
+
+    const { success, dismiss } = useFeedbackMessage()
+
+    const onDismiss = (): void => { dismiss('messageKey') }
+
+    const onClick = (): void => {
+      success({
+        extra: <Button size={Size.Small} onClick={onDismiss}>Dismiss</Button>,
+        key: 'messageKey',
+        message,
+        sticky: true,
+      })
+    }
+
+    return <Button onClick={onClick}>Click me to show a Feedback Message</Button>
+  }],
+}
+
+export const FeedbackMessageAtTheBottomOfThePage: Story = {
+  decorators: [(_Story, context) => {
+    const { info, dismiss } = useFeedbackMessage()
+
+    const { message } = context.args
+
+    return (
+      <div style={{ display: 'flex', gap: '4px' }}>
+        <Button
+          hierarchy={Hierarchy.Neutral}
+          onClick={() => info({ message, position: FeedbackMessagePosition.Bottom, duration: 10 })}
+        >
+Open
+        </Button>
+        <Button
+          hierarchy={Hierarchy.Neutral}
+          onClick={() => dismiss()}
+        >
+Dismiss
+        </Button>
+      </div>
+    )
+  }],
+}
+
+export const ReplaceFeedbackMessages: Story = {
+  decorators: [() => {
+    const { loading, success } = useFeedbackMessage()
+
+    const [isLoading, setIsLoading] = useState(false)
+
+    const MESSAGE_KEY = 'MESSAGE_KEY'
+    const DURATION = 5
+
+    const onCreateLoadingMessage = (): void => {
+      loading({
+        key: MESSAGE_KEY,
+        message: 'Loading. Please wait...',
+        duration: DURATION,
+      })
+
+      setTimeout(() => setIsLoading(false), 1000 * DURATION)
+      setIsLoading(true)
```

Comment:
```suggestion
      setTimeout(() => setIsLoading(true), 1000 * DURATION)
```
can't we just do it this way?

Reply:
Uh, no, I need to set `isLoading` to true right away, and reset it after 5 seconds.

This is necessary to trigger the `success` status only if the `isLoading` is true (which means: I'm showing the FeedbackMessage with status `loading`).

Reply:
Uh, no, I need to set `isLoading` to true right away, and reset it after 5 seconds.

This is necessary to trigger the `success` status only if the `isLoading` is true (which means: I'm showing the FeedbackMessage with status `loading`).